README.md

```diff
@@ -5,9 +5,69 @@
 Here, we explore Smoothed Particle Hydrodynamics (SPH) with two simulators available via **Inductiva API** - [SplishSplash](https://github.com/inductiva/inductiva/wiki/SPlisHSPlasH) and [DualSPHysics](https://github.com/inductiva/inductiva/wiki/DualSPHysics). 
 
 In this repository, we make a tour of **Inductiva API**:
+- [Run a simulation](README.md#running-a-simulation);
 - [Create a scenario](scenario/README.md);
 - [Generate a dataset via scenario](dataset/README.md).
 
+### Running a Simulation
+
+For **power users** who know how to configure a specific simulator and prepare the respective simulation files, **Inductiva API** provides a simple interface to simulate on a large scale. Here, we demonstrate how to use two SPH simulators in **Inductiva API**.
+
+##### SplishSplash
+
+SplishSplash is simple to run and the simulator is simply configured via a few configuration files. The required one is a `.json` file that contains the parameters of the simulation. The others are extra input files that determine the geometry and properties of the objects and fluids in the simulation.
+
+```python
+import inductiva
+
+input_dir = inductiva.utils.files.download_from_url(
```

Comment:
But this demos should be as simple as possible.
If we can avoid importing inductiva utils it would be better.
If the files are in the repo, then the user will just need to point to them (or we can just have a relative path somewhere).

Reply:
Will do that. However, keeping the download for now. We will see how the documentation evolves before removing.